lib/percy/client/resources.rb

```diff
@@ -43,6 +43,21 @@ def serialize
         }
       end
 
+      def ==(other)
+        self.class === other &&
```

Comment:
No I actually meant case equality operator. But I guess that can be replaced with `.is_a?`: https://github.com/bbatsov/ruby-style-guide#no-case-equality